addons/l10n_ch/models/account_invoice.py

```diff
@@ -278,6 +291,15 @@ def split_total_amount(self):
         """
         return float_split_str(self.amount_residual, 2)
 
+    def action_invoice_print(self):
+        '''
+        Override allowing for the print to be the one including the QR bill rather than the classic invoice.
+        '''
+        if any(not move.is_invoice(include_receipts=True) for move in self):
+            raise UserError(_("Only invoices could be printed"))
+        self.filtered(lambda inv: not inv.is_move_sent).write({'is_move_sent': True})
+        return self.env.ref('l10n_ch.invoice_with_qr').report_action(self)
```

Comment:
If this is the expected behavior, you should remove `l10n_ch.invoice_with_qr` and add the QR everytime when printing the invoice report